index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To-Dos App</title>
+</head>
+  <body>
+      <h1>To-Dos</h1>
+      <form action="#">
```

Comment:
@jjw18th Add 2 dumy to-do items - one which indicates an incomplete item and another which indicates a completed item. The completed item's text should be strikethrough and its checkbox should be checked.

Reply:
Done